dashboard/src/components/widget/description/che-description.styl

```diff
@@ -0,0 +1,18 @@
+$description-text-color = #6e6e6e
+$description-border-color = #cbcbcb
+$description-background-color = #EDEDED
```

Comment:
ok seems #CBCBCB color is also used there https://github.com/eclipse/che/blob/a8dfb6bf541940c636590119ef2fb45c1fa659c9/dashboard/src/components/widget/empty-state/che-empty-state.styl#L2